horace_core/sqw/@sqw/generate_cube_sqw.m

```diff
@@ -0,0 +1,65 @@
+function out = generate_cube_sqw(shape)
+    % Generate a cube of SQW data FOR TESTING CUT/SYMMETRISE
+    % The data will be invalid in most circumstances
+    out = sqw();
+
+    if isscalar(shape) && isnumeric(shape)
+        ndim = 4;
+        npix = shape^ndim;
+        coords = zeros(9, npix);
+
+        if mod(shape, 2) == 0
+            col = [-shape/2 + 0.5:shape/2 - 0.5];
+        else
+            col = [-floor(shape/2):floor(shape/2)];
+        end
+
+        for i = 1:ndim
+            coords(i, :) = repmat(repelem(col, shape^(i-1)), 1, shape^(ndim-i));
```

Comment:
Be really good to have a description in words of what this does, may be with an example.